### PR TITLE
fix minor typos

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by duglin** [comment]
can I get a couple of LGTMs on this before I merge, just as a sanity check ?

File: spec.md [hunk 1]
```diff
@@ -441,7 +441,7 @@ be of interest.
 Each specification that defines how to serialize a CloudEvent will define how
 extension attributes will appear.
 
-Extension attribtue MUST be serialized using the same general pattern as all
+Extension attribute MUST be serialized using the same general pattern as all
```
Comment (deissnerk) [inline spec.md]: Isn't it plural?
attribute**s**

Comment (duglin) [reply spec.md]: this is why I ask for reviews!!  :-)  fixed

**Comment by markpeek** [comment]
LGTM

**Comment by deissnerk** [comment]
LGTM

**Comment by duglin** [comment]
thanks guys.
Since this is just minor typos I'm merging.

**Pull request merged**